src/ui/theme/components/button.ts

```diff
@@ -53,108 +27,76 @@ const commonToggleButtonStyle = {
  ** Disabled
  ** Hovered
  */
-function variantSolid(props: any) {
+const variantSolid = (getColor: GetColor) => (props: any) => {
   const { colorScheme } = props;
+  // dynamic colors based on colorMode
+  let bgColor = getColor('ui.white', 'ui.black', 'ui.sepia');
+  let color = getColor('gray.800', 'ui.white', 'gray.800');
 
+  // style stays the same when it's being labeled with a colorScheme prop.
   switch (colorScheme) {
-    case 'sepia':
-      return {
-        ...commonToggleButtonStyle,
-        bg: 'ui.sepia',
-        color: 'gray.800',
-        _focus: {
-          bg: 'ui.sepia',
-          color: 'ui.black',
-        },
-        _active: {
-          bg: 'ui.sepia',
-          color: 'ui.black',
-        },
-        _hover: {
-          bg: 'ui.sepia',
-          _disabled: {
-            bg: 'ui.sepia',
-          },
-        },
-        _checked: {
-          bg: 'ui.sepia',
-          color: 'ui.black',
-        },
-        _disabled: {
-          bg: 'ui.sepia',
-        },
-      };
+    case 'light':
+      bgColor = 'ui.white';
+      color = 'gray.800';
+      break;
```

Comment:
Do we need this switch statement if we are using the above `getColor` function? Doesn't that do the same thing?

Reply:
The Day/Sepia/Night buttons behave a little different from the rest of the buttons, their colors should be consistent across different color modes (Per old design).
So I was thinking to use the `colorScheme` props to override the default setting(dynamic color). I am not sure if this is a good way to handle this.